web/src/lib/components/shared-components/album-selection-modal.svelte

```diff
@@ -13,15 +13,9 @@
   let search = '';
 
   const dispatch = createEventDispatcher<{
-    newAlbum: {
-      albumName: string;
-    };
-    album: {
-      album: AlbumResponseDto;
-    };
-    newSharedAlbum: {
-      albumName: string;
-    };
+    newAlbum: string;
```

Comment:
(album, { shared: true })

Reply:
I kept only `newAlbum` they both do the same thing